api/capi/include/nnstreamer.h

```diff
@@ -129,6 +129,7 @@ typedef enum {
   ML_NNFW_TYPE_VIVANTE = 7,           /**< VeriSilicon's Vivante. (Since 6.0) */
   ML_NNFW_TYPE_EDGE_TPU = 8,          /**< Google Coral Edge TPU (USB). (Since 6.0) */
   ML_NNFW_TYPE_ARMNN = 9,             /**< Arm Neural Network framework (support for caffe and tensorflow-lite). (Since 6.0) */
+  ML_NNFW_TYPE_SNPE = 10,
```

Comment:
let's add a detail comment like other types.

Reply:
Thanks!  I'll add it.